test/unit/cff_parser_spec.js

```diff
@@ -88,23 +88,6 @@ describe('CFFParser', function() {
     expect(names[0]).toEqual('ABCDEF+Times-Roman');
   });
 
-  it('sanitizes name index', function() {
-    var index = new CFFIndex();
-    index.add(['['.charCodeAt(0), 'a'.charCodeAt(0)]);
-
-    var names = parser.parseNameIndex(index);
-    expect(names).toEqual(['_a']);
-
-    index = new CFFIndex();
```

Comment:
Nit: `CFFIndex` needs to be removed from the `import`s at the top of the file, as pointed out in https://travis-ci.org/mozilla/pdf.js/builds/289770606#L531.